### DISCUSSION TITLE
[科技新聞] AI救地球！AI 保護環境還比人類快 3000 倍

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS3073-C/Python-Community-GS3073-C#311] [General]
![maxresdefault-14](https://user-images.githubusercontent.com/91121500/168731179-6f0f318d-d790-4404-b8ac-0eb85522f223.jpg)

氣候變遷是人類面臨的最大挑戰之一，全球變暖、冰川消融、物種滅絕、海平面上升到極端天氣事件頻發等，可挽回的時間越來越緊迫。

這將對上萬種物種產生毀滅性的影響，常見、珍稀和瀕危物種都難逃一劫。而許多動物保育或環保團體正在抓緊時間，記錄下野生動物的姿態、習性乃至等聲音等資料。然而以人力一項項辨識分析巨量的資料往往是事倍功半、耗時過久，此時的優勢變展現出來：經訓練的辨識資料的時間比人類少了 1/3000！等於人類辨識完一張相片的時間，已經辨識完 3000 張了。

![Wildlife_Insights](https://user-images.githubusercontent.com/91121500/168730814-9d95ffa4-8ee9-4264-9e68-618fb697e073.gif)
來源：https://blog.google/products/earth/ai-finds-where-the-wild-things-are/

為了處理這些人力無法負擔的百萬張照片，Google 成立了「Wildlife Insights」平台，並訓練了專責辨識照片的 。根據介紹，Google 用了 870 萬張動物圖像去訓練 ，這比全世界博物館中的動物標本總和都還多。

任何人只要將相片上傳到此平台，就會自動分析識別照片中有沒有拍到動物、是什麼動物。

成果：過去若以人力一張張辨識照片影像的話，每個小時最極限也只能處理 1,000 張照片。但如果用 Google Wildlife Insights影像辨識分析的速度可達3,000 倍以上，等於每小時能處理多達3600萬張照片。

**Comment by EricAgumonJou** [comment]
為了保育面臨絕種危機的座頭鯨，Google 在東京舉行的「Solve with AI」論壇上，公開了 Google 與美國國家海洋暨大氣總署（NOAA）合作訓練神經網路模型的，可自動偵測辨識座頭鯨的聲音，並進一步定位其所在地。

**Comment by EricAgumonJou** [comment]
要從這些錄音中辨識出座頭鯨的聲音並不簡單，水下錄音機會錄下各式各樣的聲音，而雨聲或是船隻的噪音，也常混淆判斷。並且座頭鯨的叫聲特別難分辨，不像其他鯨類會發出固定的叫聲，座頭鯨的叫聲複雜且多變，故加深了辦識難度。

座頭鯨的照片如下：
![160e352270ee8b37ba5f843a5dd9ebe1](https://user-images.githubusercontent.com/91121500/168731905-86830916-5f97-4526-a82a-fc6577f64196.jpg)

**Comment by EricAgumonJou** [comment]
Google AI 將這些水下聲音轉視覺化，以頻譜表示，不斷標記頻譜資料訓練，讓 AI 更精準的標記出錄音中的座頭鯨聲音。
現在則已經可以根據錄音中的座頭鯨聲音，在地圖上標記出他們出現的時間地點，甚至得知牠們的遷徙途徑，用於保育和研究。

**Comment by lx3211** [comment]
人類之前用科技摧毀環境，現在要用科技來保護環境了

**Comment by xCk27x** [comment]
但稀有物種的資料太少
AI模型的訓練若不夠
感覺高機率會造成誤判

**Comment by NcuJerryKao** [comment]
很大筆的資料 交給電腦 就是快啊

**Comment by Ben6uo** [comment]
但如果是新物種的話AI會怎樣做?

**Comment by BingRong1217** [comment]
沒想到AI竟然也可以做到這些
